app/upgrades/v5/upgrades.go

```diff
@@ -253,3 +260,26 @@ func UpdateIBCDenomTraces(ctx sdk.Context, transferKeeper ibctransferkeeper.Keep
 func equalTraces(dtA, dtB ibctransfertypes.DenomTrace) bool {
 	return dtA.BaseDenom == dtB.BaseDenom && dtA.Path == dtB.Path
 }
+
+
+// UpdateSlashingParams updates the Slashing params (SignedBlocksWindow) to
+// increase to keep the same wall-time of reaction time, since the block times
+// are expected to be 67% shorter.
+func UpdateSlashingParams(ctx sdk.Context, xk slashingkeeper.Keeper, pk paramskeeper.Keeper) {
```

Comment:
you can just use the slashing keeper's `Get/SetParams` to update the value instead of passing the params keeper

Reply:
done in [b5bfcce](https://github.com/tharsis/evmos/pull/699/commits/b5bfcce1069325b3d44cb512d3b8dff0bf192a1a)